lod/mca-css.md

```diff
@@ -0,0 +1,130 @@
+
+## Create a Cloud Slice Application 
+
+An Azure enterprise application needs to be created for Cloud Slice. The application will be used for management of subscriptions by Skillable Studio. 
+
+1. In the Azure portal, navigate to Azure Active Directory. 
+1. Select **Enterprise Applications**.
+1. Select **+ New Application**.
+1. Select **+ Create your own application**.
+1. Enter a name for the application. 
+1. Select the option **Integration any other application you don't find in the gallery (Non-gallery)**.
+1. Select **Create**.
+1. Navigate to **App Registrations** in Azure AD and select the application you just created.
+1. Select **Certificates & Secrets** and generate a client secret for Skillable Studio to use, taking note of the value of the secret you generate. Store this information in a secure location. 
+ 
+## MCA CSS Pool Creation
+
+Configuration of the MCA must be completed in the Azure portal first, and then a subscription pool must be created in Skillable Studio. 
+
+### Azure Billing Scope and IAM Configuration 
+
+1. Sign into the MCA tenant with the local global admin account you created, and navigate to the **Cost Management + Billing** Section.
+
+1. In the Azure portal, navigate to **Cost Management + Billing**. 
+1. Select **Billing Scopes**.
+1. Select the **Billing Account** for the MCA you've just created.
+ 
+    ![Select Billing Account](images/select-billing-account.png)
+ 
+1. Navigate to **Access Control (IAM)** and add the cloud slice application created in previous steps, as well as any users that you need to have **management access to the billing scope**.
+
+    ![Navigate to Access Control](images/navigate-to-access-control.png)
+ 
+### Configure Tenant Root Group
+
+The enterprise application needs to be configured with the **Owner** role in the Tenant Root Group. 
+
+1. In the Azure portal, search for **Management Groups**. 
+
+1. Select **Tenant Root Group**. 
+
+    ![Tenant Root Group](images/tenant-root-group.png)
+
+1. On the next screen, select **Access control (IAM)**.
+
+    ![Access Control (IAM)](images/access-control-iam.png)
+
+1. Select **+Add**, and then choose **Add role assignment**. 
+
+    ![Add button](images/add-button.png)
+
+1. Select the **Owner role** at the top of the list. If the role is not listed at top, search for **Owner**. 
+
+    ![Owner role](images/owner-role.png)
+
+1. After selecting the Owner role, click **Next**. 
+
+1. Select **+Select Members**. 
+
+1. Search for the name of the enterprise application that was created in previous steps. 
+
+1. Click the application and click **Select**. 
+
+1. Select **Next**. 
+
+1. On the **Conditionals(optional)** screen, select **Next**.  
+
+1. Confirm the role, scop and member for the role assignment and select **Review + assign**.
```

Comment:
Is "scop" supposed to be scope?

Reply:
Good catch! I have updated that to say "scope" now.